src/Tickets/Commerce/Gateways/Stripe/Merchant.php

```diff
@@ -192,6 +190,92 @@ public function save_signup_data( array $signup_data ) {
 		return update_option( $this->get_signup_data_key(), $signup_data );
 	}
 
+	/**
+	 * Returns the list of enabled payment method types for the Payment Element, or the Card type
+	 * for the Card Element.
+	 *
+	 * @since TBD
+	 *
+	 * @return string[]
+	 */
+	public function get_payment_method_types( $fallback = false ) {
+
+		if ( $fallback || Settings::CARD_ELEMENT_SLUG === tribe_get_option( Settings::$option_checkout_element ) ) {
+			return [ 'card' ];
+		}
+
+		return tribe_get_option( Settings::$option_checkout_element_payment_methods, [ 'card' ] );
+	}
+
+	/**
+	 * Query the Stripe API to gather information about the current connected account.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $client_data connection data from the database
+	 *
+	 * @return array
+	 */
+	public function check_account_status( $client_data = [] ) {
+
+		if ( empty( $client_data ) ) {
+			$client_data = $this->to_array();
+		}
+
+		$return = [
+			'connected'       => false,
+			'charges_enabled' => false,
+			'errors'          => [],
+			'capabilities'    => [],
+		];
+
+		if ( empty( $client_data['client_id'] )
+			 || empty( $client_data['client_secret'] )
+			 || empty( $client_data['publishable_key'] )
+		) {
+			return $return;
+		}
+
+		$account_id = urlencode( $client_data['client_id'] );
+		$url        = '/accounts/{account_id}';
+		$url        = str_replace( '{account_id}', $account_id, $url );
```

Comment:
Could we use `sprintf()` here?
```suggestion
		$url        = sprinf( '/accounts/%s', $account_id );
```